compiler/tests/fixtures/demos/uniswap.fe

```diff
@@ -0,0 +1,359 @@
+contract UniswapV2Factory:
+    fee_to: address
+    fee_to_setter: address
```

Comment:
Hmm, I did not notice that we don't support visibility specifiers. So by default, every variable is private?

Reply:
Yeah, for this I was thinking that we should just manually implement `fee_to` and `fee_to_setter` public getter functions. It might be better, though, to add visibility modifiers for fields to the list of "nice to have features" and if we can't finish support for it by April, we would just use the public functions.

On a side note, we could add this to the list of things that could be implemented using a desugaring pass. 

i.e.

```
pub fee_to: address
```

becomes

```
fee_to: address

pub def fee_to() -> address:
  return `self.fee_to`
```

Reply:
>On a side note, we could add this to the list of things that could be implemented using a desugaring pass.

I like that idea :+1: